src/main/webapp/index.html

```diff
@@ -5,40 +5,66 @@
+
+      <ul class="navbar-nav">
+        <li class="nav-item"><a class="nav-link" href="index.html">Home</a></li>
+      </ul>
+    </nav>
     <!--if the page browses slowly-->  
     <p id="loading">L O A D I N G</p > 
 
     <div id="login_page"></div>
     <div id="index_content">
-      <div id="about">
-        <text id="about_text">
-          Get started with the <text id="gpostcard">GPostCard!</text><br/>
-          <br/>Type in the text for your postcard.<br/> 
-          Tick the location checkbox if you want to share your travels!<br/>
-          We will generate a perfect postcard for you.</br>
-          You just need to SEND it! <br/>
-        </text>
+      <div class="inline" id="about">
+        <main class='container'>
+          <div class='env'>
+            <label for='open-env'>
+            <input type='checkbox' id='open-env' />
+            <label class='top' for='open-env'>
+              <br/>
+              <text id="open">Open me!</text>
+            </label>
+            <div class='content'>
+              <text id="about_text">
+                Get started with the <text id="gpostcard">GPostCard!</text> <br/>
+                Type in the text for your postcard. <br/>
+                Tick the location checkbox if you want to share your travels! <br/>
+                We will generate a perfect postcard for you. <br/>
+                You just need to SEND it! <br/>
+              </text>
+            </div>
+            <div class='rest'></div>
+            </label>
+          </div>
+        </main>
       </div>
-      <form id="input_form">
-        <input type="text" class="input_text" id="input_text" name="input_text" onclick="empty()" value="your greating"> 
-        <label class="container">Tick to share location
-          <input type="checkbox" id="location_checkbox" name="location_checkbox" onclick="getUserLocation()" value="none">
-          <span class="checkmark"></span>
-        </label>
-        <!--<input class="checkbox" type="checkbox" id="location_checkbox" name="location_checkbox" onclick="getUserLocation()" value="none"\>
-        <text class="checkbox">Tick to share location</text>-->
-        <br/>
-     </form>
-     <button onclick="generatePostcard()" id="submit_input">Submit</button>
-     <button onclick="record()" id="recording">Start Recording</button>
-     <div id="recording-message"></div>
+      <div class="inline">
+        <form id="input_form">
+          <input type="text" class="input_text" id="input_text" name="input_text" placeholder="Enter your postcard text.." required="True"> 
+          <label class="container"> Tick to share location
+            <input type="checkbox" id="location_checkbox" name="location_checkbox" onclick="getUserLocation()" value="none">
+            <span class="checkmark"></span>
+          </label>
+          <br/>
+        </form>
+        <button onclick="generatePostcard()" id="submit_input">Generate</button>
```

Comment:
You started to separate words in class and id names with hyphen, please keep it consistent.
Also, in order to improve understanding and scannability, hyphen separation is recommended instead of underscores.